gremlin-go/driver/graphBinary.go

```diff
@@ -983,6 +1028,8 @@ func (serializer *graphBinaryTypeSerializer) getSerializerToWrite(val interface{
 		return &graphBinaryTypeSerializer{dataType: PType, writer: pWriter, logHandler: serializer.logHandler}, nil
 	case textP, TextPredicate:
 		return &graphBinaryTypeSerializer{dataType: TextPType, writer: textPWriter, logHandler: serializer.logHandler}, nil
+	case *Binding:
```

Comment:
If you have a pointer and non-pointer support in the function, should you accept both here?

Reply:
Or remove support for non-pointers in the function?

Reply:
What do you mean here?